meteor/sofie-server/data-access/repositories/test/mongo-test-database.ts

```diff
@@ -30,85 +26,46 @@ export class MongoTestDatabase {
 		}
 	}
 
-	public getValidObjectIdString(base: string): string {
-		const twelveChar = base.length >= 12 ? base.substring(0, 12) : this.fillTo12Chars(base).substring(0, 12)
-		return this.convertToHex(twelveChar)
-	}
-
-	private fillTo12Chars(base: string): string {
-		const missingLength = 12 - base.length + 2
-		const multiplier = this.replaceAll('1' + Array<number>(missingLength).fill(0).join(), ',')
-		return base + Math.floor(Math.random() * +multiplier)
-	}
-
-	// Can and should be replaced with 'string.replaceAll(...)', when the project is updated to target Es2021.
-	private replaceAll(target: string, searchFor: string, replaceWith: string = ''): string {
-		while (target.includes(searchFor)) {
-			target = target.replace(searchFor, replaceWith)
-		}
-		return target
+	public getDatabase(): Db {
+		return this.client.db(this.mongoServer.instanceInfo!.dbName)
 	}
 
-	private convertToHex(base: string): string {
-		let hex = ''
-		for (let i = 0; i < base.length; i++) {
-			const charCode = base.charCodeAt(i)
-			const hexValue = charCode.toString(16)
-			hex += hexValue.padStart(2, '0')
+	public async populateDatabaseWithActiveRundowns(rundowns: MongoRundown[]): Promise<void> {
+		const db: Db = this.getDatabase()
+		for (const rundown of rundowns) {
+			await db.collection('rundowns').insertOne(rundown)
+			await db.collection('rundownPlaylists').insertOne({
+				externalId: rundown.name,
+				activationId: 'activated',
+			})
```

Comment:
I guess @RasmusAlbrektsen should be aware that this exist so he can change it with his changes to how activation is going to work

Reply:
Same goes for `populateDatabaseWithInactiveRundowns`

Reply:
I've already caught most of it in my PR, but i'll look it through when i merge these changes into my branch again  :-)

Reply:
I notified him about this change in person, before commiting it.